### ISSUE TITLE
"+" (Add Nuget Package button) which is at the bottom of the left pane is not showing the tooltip once user traverse the same using keyboard.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#266]
### Steps to Reproduce

1.Launch Xamarin Workbook application and select platform and create a new workbook.
2. Navigate to "+" button which is at the bottom of the left pane of the workbooks window using keyboard. and observe the tooltip.

### Expected Behavior
Tooltip should be displayed once keyboard focus is traversing the "+" button.

### Actual Behavior
Tooltip is not getting appeared once user traverse using keyboard.
Note: This "+" (Add Nuget Package button) is sometimes accessible and sometimes not.
So this is inconsistent behaviour of the keyboard for this "+" button.

### User Impact:
Keyboard user won't be able to know the purpose of the button, until its tooltip is not defined correctly.

### Environment

MAC: macOS High Sierra [10.13.4]
Workbook - 1.4 (master/b3f92f9)We saved this conversation. You'll see it soon in the Conversations tab in Skype for Business and in the Conversation History folder in Outlook. 

### Log File

![kb9_leftpane_addpackagebutton](https://user-images.githubusercontent.com/22956130/39857404-c3bdcb5c-5422-11e8-861a-fd6677101b4d.png)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS4.2.1
Xamarin-Workbook-MAC
Wipro

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1816
